images.go

```diff
@@ -0,0 +1,52 @@
+package shopify
+
+import "time"
+
+// Images is a collection of Images
+type Images []Image
+
+// Image is an image used by the product
+type Image struct {
```

Comment:
this has the wrong values for a generic image like the one on a collection, check the docs for the actual values